### DISCUSSION TITLE
Pop-up numbers in selection speed?

### DISCUSSION THREAD

**Discussion by summeremacs** [meow-edit/meow#312] [Q&A] [answered]
I did have another question (I hope I'm not annoying people):

When I select something with, for instance 't' for 'till', I get a bunch of numbers to go to if I want. But it's super fast on my screen. I'm wondering if I can make it last like 5 seconds or something instead of just half a second? Is there a way to do that? I didn't see it in the help at all. I was hoping for the pop-ups to remain until a choice is made, like with avy.

**Comment by Linerre** [comment, marked as answer]
You are welcome to ask any questions about meow.

Regarding your question, try to set this variable to a larger number:
```
;; in your meow cconfig
(setq meow-expand-hint-remove-delay 5)
```

You can find a bunch of variables that you can use to customize your meow behavior:
https://github.com/meow-edit/meow/blob/master/CUSTOMIZATIONS.org

**Comment by eshrh** [comment]
Making pop-ups remain until a choice is made or another key is pressed is definitely an idea, but like linerre mentioned you can achieve this by just setting meow-expand-hint-remove-delay to an absurdly large number.

Do note though, that you can still press the 1-10 numeric expand keys even after the hints disappear, it's just that you won't be able to see where you're going.

**Comment by summeremacs** [comment]
Thank you both so much. It's exactly what I was looking for. 🙃 (I set it to 10). I'm still new to editing in this way so sometimes it takes me a while to figure out what I'm doing and where to go (my fingers move faster than my thoughts sometimes). This is perfect.
